### PR TITLE
Update release-6.3.0.md

### PR DESCRIPTION
**Pull request by solotzg**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [ ] v6.4 (TiDB 6.4 versions)
- [x] v6.3 (TiDB 6.3 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by shichun-0415** [comment]
This note seems an open issue found in 6.3, not an improvement made in 6.3. I guess we will fix it in future versions. How about we adding the bug fix to the release notes of the future patch and minor releases?

**Comment by solotzg** [comment]
> This note seems an open issue found in 6.3, not an improvement made in 6.3. I guess we will fix it in future versions. How about we adding the bug fix to the release notes of the future patch and minor releases?

Issue https://github.com/pingcap/tiflash/issues/6075 is caused by that someone use a too old platform to deploy TiFlash v6.3.0. We need to make an explanation that TiFlash reqiures some CPU instruction sets since v6.3.0.

**Comment by shichun-0415** [comment]
> We need to make an explanation that TiFlash reqiures some CPU instruction sets since v6.3.0.

Let's put the statement in Compatibility Changes > Others. https://github.com/pingcap/docs/pull/10631/files#diff-577672dd123c42d722caad525849ff7ee636eab7a8ca14597c7e674a93cbc04fR246
image

**Comment by solotzg** [comment]
> > We need to make an explanation that TiFlash reqiures some CPU instruction sets since v6.3.0.
> 
> Let's put the statement in Compatibility Changes > Others. 

Done.

File: releases/release-6.3.0.md [hunk 1]
```diff
@@ -248,6 +248,7 @@ In v6.3.0-DMR, the key new features and improvements are as follows:
 * Log backup supports GCS and Azure Blob Storage as backup storage.
 * Log backup is now compatible with the `exchange partition` DDL.
 * The SQL statement `ALTER TABLE ...SET TiFLASH MODE ...` previously used for enabling [fastscan](/develop/dev-guide-use-fastscan.md) is deprecated, and replaced by the system variable [`tiflash_fastscan`](/system-variables.md#tiflash_fastscan-new-in-v630). When you upgrade from v6.2.0 to v6.3.0, all FastScan settings in v6.2.0 will become invalid, but will not affect the normal reading of data. In this case, you need to configure the variable [`tiflash_fastscan`](/system-variables.md#tiflash_fastscan-new-in-v630) to enable or disable FastScan. When you upgrade from an earlier version to v6.3.0, the FastScan feature is not enabled by default for all sessions to keep data consistent.
+* To deploy TiFlash under Linux with AMD64 architecture, the CPU must support AVX2 instruction sets, use `cat /proc/cpuinfo | grep avx2` to confirm there is output. TiFlash's vectorization engine needs CPU instruction sets to perform a better effect.
```
Comment (shichun-0415) [inline releases/release-6.3.0.md:251]: `TiFlash's vectorization engine needs CPU instruction sets to perform a better effect.` Can's see any relationship between this sentence and the other two.
```suggestion
* To deploy TiFlash under the Linux AMD64 architecture, the CPU must support AVX2 instruction sets. Use `cat /proc/cpuinfo | grep avx2` to confirm that there is output. TiFlash's vectorization engine needs CPU instruction sets to perform a better effect.
```

**Review by zanmato1984 — APPROVED** [review]
LGTM

**Review by zanmato1984 — APPROVED** [review]
LGTM

File: releases/release-6.3.0.md [hunk 2]
```diff
@@ -248,6 +248,7 @@ In v6.3.0-DMR, the key new features and improvements are as follows:
 * Log backup supports GCS and Azure Blob Storage as backup storage.
 * Log backup is now compatible with the `exchange partition` DDL.
 * The SQL statement `ALTER TABLE ...SET TiFLASH MODE ...` previously used for enabling [fastscan](/develop/dev-guide-use-fastscan.md) is deprecated, and replaced by the system variable [`tiflash_fastscan`](/system-variables.md#tiflash_fastscan-new-in-v630). When you upgrade from v6.2.0 to v6.3.0, all FastScan settings in v6.2.0 will become invalid, but will not affect the normal reading of data. In this case, you need to configure the variable [`tiflash_fastscan`](/system-variables.md#tiflash_fastscan-new-in-v630) to enable or disable FastScan. When you upgrade from an earlier version to v6.3.0, the FastScan feature is not enabled by default for all sessions to keep data consistent.
+* To deploy TiFlash under the Linux AMD64 architecture, the CPU must support AVX2 instruction sets. Use `cat /proc/cpuinfo | grep avx2` to confirm that there is output. TiFlash's vectorization engine needs CPU instruction sets to perform a better effect.
```
Comment (solotzg) [inline releases/release-6.3.0.md:251]:
```suggestion
* To deploy TiFlash under the Linux AMD64 architecture, the CPU must support AVX2 instruction sets. Use `cat /proc/cpuinfo | grep avx2` to confirm that there is output. By using such CPU instruction sets, TiFlash's vectorization engine can deliver better performance.
```

**Review by shichun-0415 — APPROVED** [review]
LGTM

**Pull request merged**